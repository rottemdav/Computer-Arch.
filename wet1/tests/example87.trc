32 4 0 3 local_history global_tables not_using_share
0x73fa564 T 0xd578679c
0x10f8 N 0xac39e0a0
0xd59bd244 T 0x29c8
0x618f63c8 T 0xbec
0x04c8 T 0x7099f8
0x618f63c8 N 0xbec
0x71274 N 0xce44624
0x73c3c T 0x800400
0x9fc4 T 0xc64924
0xb3068 N 0xb075c
0xf4084da4 N 0xe034
0xa1cc T 0xc6c
0x80c4 N 0xdaf384
0xa968 T 0xa4f0
0xb274 N 0xddf8