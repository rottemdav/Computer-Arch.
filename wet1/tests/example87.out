0x73fa564 N 0x73fa568
0x10f8 N 0x10fc
0xd59bd244 N 0xd59bd248
0x618f63c8 N 0x618f63cc
0x4c8 T 0xbec
0x618f63c8 T 0x7099f8
0x71274 N 0x71278
0x73c3c N 0x73c40
0x9fc4 T 0x29c8
0xb3068 N 0xb306c
0xf4084da4 N 0xf4084da8
0xa1cc N 0xa1d0
0x80c4 T 0xc64924
0xa968 T 0xb075c
0xb274 T 0xce44624
flush_num: 11, br_num: 15, size: 1152b
